README.md

```diff
@@ -1,242 +1,278 @@
-# Quint Lang
+<div align="center">
+
+<!-- Title -->
+<img src="./logos/quint-logo.png" alt="Quint Lang Logo" width="120"/>
+<h1>Quint Lang</h1>
+
+<!-- Menu -->
+<p>
+    <a href="#installation">Installation</a> •
+    <a href="./doc#readme">Documentation</a> •
+    <a href="#community">Community</a>
+</p>
+
+<!-- Badges -->
+<p>
+    <a href="https://github.com/informalsystems/quint/actions">
+        <img
+            src="https://github.com/informalsystems/quint/actions/workflows/main.yml/badge.svg"
+            alt="build badge">
+    </a>
+    <a href="https://marketplace.visualstudio.com/items?itemName=informal.quint-vscode">
+        <img
+            src="https://img.shields.io/visual-studio-marketplace/v/informal.quint-vscode?color=10b0f2&label=VSCode"
+            alt="Visual Studio Marketplace Version">
+    </a>
+    <a href="https://www.npmjs.com/package/@informalsystems/quint">
+        <img
+            src="https://img.shields.io/npm/v/@informalsystems/quint"
+            alt="npm (scoped)">
+    </a>
+</p>
+</div>
+
+
+## Overview
+
+Quint is a modern specification language that is a particularly good fit for
+distributed systems, such as blockchain protocols, distributed databases, and
+p2p protocols. Quint combines the robust theoretical basis of the [Temporal
+Logic of Actions (TLA)][TLA] with state-of-the-art static analysis and
+development tooling.
+
+### Example code in Quint :mrs_claus: :gift: :santa:
+
+``` bluespec
+module secret_santa {
+  import basicSpells.* from "../../spells/basicSpells"
+  import commonSpells.* from "../../spells/commonSpells"
+
+  const participants: Set[str]
+
+  /// get(recipient_for_santa, S) is the recipient for secret santa S
+  var recipient_for_santa: str -> str
```

Comment:
Counterpoint: itIsHarderToReadWordsWhenTheyAreAllMashedTogether :stuck_out_tongue_closed_eyes: 

I prefer snakes, but I'll defer to @bugarela on this to, who first opted for this naming.

Reply:
I think this problem is undecidable :laughing: there is no way to please everyone here, and considering we will have a completely mixed audience, I chose snake case because that is easier to read in general. Of course, people who prefer camel case won't agree, but there's no way to please everyone (on the example - quint itself won't force a style ofc).